app/screens/MachineDetails.js

```diff
@@ -53,11 +53,16 @@ class MachineDetails extends Component {
         showRemoveMachineModal: false,
     }
 
-    static navigationOptions = ({ navigation }) => {
+    static navigationOptions = ({ navigation, theme }) => {
         return {
             headerLeft: <HeaderBackButton navigation={navigation} />,
             headerRight: <RemoveMachine navigation={navigation} />,
-            headerTransparent: true,
+            headerStyle: {
```

Comment:
Are we not adding back in the title `title: `${navigation.getParam('machineName')} @ ${navigation.getParam('locationName')}`` and removing: 
`       <View style={s.machineNameContainer}>
                                    <Text style={s.machineName}>{machineName}</Text>
                                    <Text style={s.locationName}>{location.name}</Text>
                                </View>
`?

Reply:
Yeah. I don't want the name in there twice, and I'd rather have it below where we can have more flexibility for formatting. And if it's just in the header, there isn't enough space. Too many machine names get cut off.

Thoughts?

Reply:
I thought we were putting the title back in the header with a sticky header. I don't understand what is different between this and what we had with `headerTransparent:true`.

Reply:
I hadn't considered the case of machine names getting cut off. I could go either way on this considering that. I just thought we were going back to the sticky header with the title at the top when I checked out this PR. My misunderstanding.

Reply:
The only difference now is when you scroll, the content goes under the header, rather than continuing to be visible.

So yeah, it’s a transparent header with no title vs a colored header with no title.

Reply:
But I’ll keep playing around with it so we can settle on something we like. 

Easy to tweak and go back and forth.